src/main/java/org/inferred/freebuilder/processor/excerpt/CheckedBiMap.java

```diff
@@ -126,9 +128,11 @@ public void addTo(SourceBuilder code) {
           .addLine("  }")
           .addLine("")
           .addLine("  @Override public V setValue(V value) {")
+          .addLine("    final K oldKey = biMap.inverse().get(value);")
```

Comment:
No "final" please, it does not match the rest of the code.

Reply:
Done, force of habit on my part.